data/json/effects_on_condition/npc_eocs/hallucination_eocs.json

```
@@ -175,26 +175,31 @@
     "type": "effect_on_condition",
     "id": "creature_hallucinations",
     "//TODO": "ideally checks is the tile visible, and then spawns hallu where player can't see it",
-    "//": "if monsters are nearby, 3/4 chance to get copy of random monster nearby and run again (with 1/2 prob) and 1/4 chance to spawns one of YOUR_FEARS. If not, spawns one of YOUR_FEARS",
-    "effect": [
+    "//": "If monsters are nearby, 3/4 chance to spawn some hallucinations of random monsters nearby and run again half the time, 1/4 chance to spawn some of YOUR_FEARS. If not, spawns some of YOUR_FEARS",
+    "condition": { "and": [ { "math": [ "rand(3)", "==", "3" ] }, { "not": { "math": [ "u_monsters_nearby()", ">", "1" ] } } ] },
```

```suggestion
    "condition": { "and": [ { "math": [ "rand(3)", "==", "3" ] }, { "math": [ "u_monsters_nearby()", "==", "0" ] } ] },
```
I don't know why i made it in this way before

---

I figured maybe the greater than 1 was if it returns the player or something odd, I'll switch the effect and false_effect around to make more sense

Idk if the default there actually does anything bc if wave_spawn_number() fails to return anything I'd of thought there would be an error but I didn't look up the implementation